app/src/main/java/net/squanchy/eventdetails/EventDetailsInjector.kt

```diff
@@ -0,0 +1,14 @@
+package net.squanchy.eventdetails
+
+import net.squanchy.injection.ActivityContextModule
+import net.squanchy.injection.ApplicationInjector
+import net.squanchy.navigation.NavigationModule
+
+internal fun eventDetailsComponent(activity: EventDetailsActivity): EventDetailsComponent {
```

Comment:
Why not in the component's file like we do elsewhere?

Reply:
good point